pkg/operator/ceph/file/controller.go

```diff
@@ -356,15 +360,28 @@ func (r *ReconcileCephFilesystem) reconcile(request reconcile.Request) (reconcil
 			errors.Wrapf(err, "invalid object filesystem %q arguments", cephFilesystem.Name)
 	}
 
+	shouldRotateCephxKeys := false
+	shouldRotateCephxKeys, err = keyring.ShouldRotateCephxKeys(cephCluster.Spec.Security.CephX.Daemon, *runningCephVersion,
```

Comment:
I think this is the same right?

```suggestion
	shouldRotateCephxKeys, err := keyring.ShouldRotateCephxKeys(cephCluster.Spec.Security.CephX.Daemon, *runningCephVersion,
```

Reply:
Yep. Same. Fixed it.